### DISCUSSION TITLE
do you have any doc more for this project

### DISCUSSION THREAD

**Discussion by chenfenggang** [flowerfine/scaleph#102] [Q&A]
I am very interest this project ,do you have any doc more? thank you

**Comment by kalencaya** [comment]
We plan to maintain docs by github wiki, you can find them [here](https://github.com/flowerfine/scaleph/wiki)

**Comment by kalencaya** [comment]
We have missed you question for a few days for ignoring discussion, issue can feed back more quickly.

**Comment by gleiyu** [comment]
@chenfenggang hi, you can also follow our github page, and the document will be updated step by step

**Reply by gleiyu** [reply]
https://flowerfine.github.io/scaleph/#/
